about.html

```diff
@@ -0,0 +1,47 @@
+<html>
+<head>
+	<title>Maria Fedorec-Kyte</title>
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,200;0,400;0,700;1,200;1,400;1,700&display=swap" rel="stylesheet">
+	<link href="style.css" type="text/css" rel="stylesheet" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+</head>
+
+<body class="about-body">
+		<div class="hamburger-menu">
+			<img src="Hamburger_icon_white.png" alt="Hamburger Icon White" height="30px">
+		</div>
+		<aside class="about">
+			
```

Comment:
i don't think these gaps either side of the aside tags are necessary but if its for personally readability then fair enough